database/seeders/LableSeeder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Lable;
+use Illuminate\Database\Seeder;
+
+class LableSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     * itemsテーブルに対していくつでも紐づく前提
+     * @return void
+     */
+    public function run()
+    {
+        // lableに紐づくitemが　　0,1,2,3のように作成されるように
+        $itemCounts = 4;
+
+        for ($i = 0; $i < $itemCounts; ++$i) {
+            Lable::factory($i)->create([
+                'item_id' => $i,
```

Comment:
```suggestion
        $itemCounts = 10;

        for ($i = 0; $i < $itemCounts; ++$i) {
            Lable::factory($i)->create([
                'item_id' => $i + 1,
```

ラベルを多めに配置した商品も作っておいた方が、フロント実装する時に役立つと思うので、いかがでしょうか？
(ラベルが多い場合、スクロールを実装する、と聞いたので。)
item_id=1→ラベル0個
item_id=2→ラベル1個
item_id=3→ラベル2個
...
item_id=10→ラベル9個